hedera-mirror-importer/src/main/resources/db/scripts/init-temp-schema.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+TEMP_SCHEMA="${HEDERA_MIRROR_IMPORTER_DB_TEMPSCHEMA:-temporary}"
```

Comment:
keep it consistent with init.sh

```suggestion
TEMP_SCHEMA="${DB_TEMPSCHEMA:-temporary}"
```